src/pages/tickets/org/orgHeader/index.tsx

```diff
@@ -201,10 +176,48 @@ export const OrgHeader = () => {
               <Label htmlFor="statusSelect">Skill</Label>
               <Skill id="statusSelect" />
             </SkillContainer>
-            <SearchWrapper>
-              <SearchBar placeholder="Search" disabled />
-              <Icon src={searchIcon} alt="Search" />
-            </SearchWrapper>
+            <SearchBar
+              name="search"
+              type="search"
+              placeholder="Search"
+              value={ui.searchText}
+              style={{
+                width: '384px',
+                height: '40px',
+                background: color.grayish.G950,
+                fontFamily: 'Barlow',
+                color: color.text2,
+                gap: '10px',
+                flex: '1 0 0',
+                display: 'flex',
+                position: 'relative',
+                marginLeft: '20px',
+                alignItems: 'center',
+                borderRadius: '6px',
+                border: 'none',
+                outline: 'none'
+              }}
+              onChange={(e: any) => {
+                ui.setSearchText(e);
+              }}
+              onKeyUp={(e: any) => {
```

Comment:
Try to avoid Inline functions, create the search function outside the component, and call it in the onKeyUp event.

Reply:
Addressed